R/find_parameters.R

```diff
@@ -224,6 +227,19 @@ find_parameters.stanreg <- function(x, ...) {
   ))
 }
 
+#' @export
+find_parameters.htest <- function(x, ...) {
+  fe <- colnames(as.data.frame(x))
+
+  cond <- fe[grepl(pattern = "^(?!(b\\[|sigma|Sigma))", fe, perl = TRUE)]
+  rand <- fe[grepl(pattern = "^b\\[", fe, perl = TRUE)]
```

Comment:
Do `htest`-objects have random effects, or even a `as.data.frame()`-method? This looks strange to me...